environments/se/core/src/test/java/org/jboss/weld/environment/se/test/builder/WeldBuilderTest.java

```diff
@@ -211,4 +211,19 @@ public void testNoBeanArchivesFound() {
         new Weld().disableDiscovery().initialize();
     }
 
+    @Test
+    public void testMultipleWeldInstancesCreated() {
```

Comment:
Hmm... I must have missed that test. Anyway I will modify the PR as you suggested, it seems pretty reasonable.